-1

**Failed Tests**:   Build 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-f7ba25/22813/summary.html
**COMMIT**: 2fdef732a3ef97c437680e65310ab2ae8fd23277
**CMSSW**: CMSSW_12_3_X_2022-03-03-1100/slc7_amd64_gcc10
**Additional Tests**: THREADING
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/37130/22813/install.sh` to create a dev area with all the needed externals and cmssw changes.



The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
 - @srimanob cms-sw/cmssw#37005
 - @tvami cms-sw/cmssw#37102
 - @swagata87 cms-sw/cmssw#37100

You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-f7ba25/22813/git-recent-commits.json
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-f7ba25/22813/git-merge-result


## Build

I found compilation error when building: 

<pre>>> Done generating edm plugin poisoned information
gmake[1]: *** [config/SCRAM/GMake/Makefile.rules:1852: CompilePython] Error 1
>> Plugins of all types refreshed.
gmake[1]: Target 'PostBuild' not remade because of errors.
gmake[1]: Leaving directory '/pool/condor/dir_96975/jenkins/workspace/ib-run-pr-tests/CMSSW_12_3_X_2022-03-03-1100'
gmake: *** [config/SCRAM/GMake/Makefile.rules:1740: src] Error 2
gmake: Target 'all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
+ eval scram build outputlog '&&' '(python3' /pool/condor/dir_96975/jenkins/workspace/ib-run-pr-tests/cms-bot/buildLogAnalyzer.py --ignoreWarning=Wdeprecated-declarations --logDir /pool/condor/dir_96975/jenkins/workspace/ib-run-pr-tests/CMSSW_12_3_X_2022-03-03-1100/tmp/slc7_amd64_gcc10/cache/log/src '||' 'true)'
++ scram build outputlog
>> Entering Package Configuration/AlCa
</pre>

--- a/Configuration/AlCa/python/autoAlca.py
+++ b/Configuration/AlCa/python/autoAlca.py
@@ -131,4 +131,6 @@ AlCaNoConcurrentLumis = [
     'PromptCalibProdSiPixelAli',       # AlignmentProducerAsAnalyzer, MillePedeFileConverter
     'PromptCalibProdBeamSpotHP',       # AlcaBeamSpotProducer
     'PromptCalibProdBeamSpotHPLowPU',  # AlcaBeamSpotProducer
+    'AlCaPCCRandom',                   # AlcaPCCIntegrator 
+    'AlCaPCCZeroBias'                  # AlcaPCCIntegrator
 ]

